src/scripts/farming/Farming.ts

```diff
@@ -1178,7 +1179,7 @@ class Farming implements Feature {
             BerryFirmness.Very_Hard,
             [
                 'This Berry is surrounded by mystery. It is rumored to be imbued with the power of all living things.',
-                'This power keeps other Berries alive for longer.',
+                'This power resets and pauses the wither timer of other Berries indefinitely.',
```

Comment:
It seems too straightforward and technical for in-game information. What do you think of this :
```suggestion
                'This power indefinitely saves other Berries from withering.',
```

Reply:
I mean, it's better than the original but I wanted some way to convey that it resets timers too as that was the sole reason I edited it.

Reply:
> Revitalizes and prevents other berries from withering

Reply:
That's perfect, thanks!